includes/blocks/example-block/save.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * See https://wordpress.org/gutenberg/handbook/designers-developers/developers/block-api/block-edit-save/#save
+ *
+ * @return {null} Dynamic blocks do not save the HTML.
+ */
+const ExampleBlockSave = () => {
+	return null;
```

Comment:
Would this work / be able to work with Server Side Render?

Reply:
Yes. This would be fine. This is standard for a dynamic block.